examples/gnrc_border_router/Makefile

```diff
@@ -11,8 +11,8 @@ RIOTBASE ?= $(CURDIR)/../..
 UPLINK ?= ethos
 
 # Check if the selected Uplink is valid
-ifeq (,$(filter ethos slip cdc-ecm wifi ethernet,$(UPLINK)))
-  $(error Supported uplinks are `ethos`, `slip`, `cdc-ecm`, `ethernet` and `wifi`)
+ifeq (,$(filter ethos slip cdc-ecm wifi ethernet tap,$(UPLINK)))
+  $(error Supported uplinks are `ethos`, `tap`, `slip`, `cdc-ecm`, `ethernet` and `wifi`)
```

Comment:
Mh... but for `ethos` the tap interface would not be reused?

Reply:
A board running `ethos` usually doesn't have any tap interfaces that could be used as an uplink instead :wink:

Reply:
Huh? `ethos` uses TAP to provide the Ethernet interface to the host.

Reply:
This is about which uplink is used by the *board* / RIOT. `netdev_tap` only exists on `native`.

Reply:
I know, but when I was reading the title and OP, I was thinking this also provides the described feature for `ethos`, as this is an instance where TAP is used with the gnrc_border_router example ...

Reply:
Heh this does actually work :smiley: 

```
Iface  5  HWaddr: 49:05  Channel: 26  Page: 0  NID: 0x23  PHY: O-QPSK 
          
          Long HWaddr: 00:04:25:19:18:01:C9:05 
           TX-Power: 0dBm  State: IDLE  max. Retrans.: 3  CSMA Retries: 4 
          AUTOACK  ACK_REQ  CSMA  L2-PDU:102  MTU:1280  HL:64  RTR  
          RTR_ADV  6LO  IPHC  
          Source address length: 8
          Link type: wireless
          inet6 addr: fe80::204:2519:1801:c905  scope: link  VAL
          inet6 addr: 2001:9e8:1411:1cf0:204:2519:1801:c905  scope: global  VAL
          inet6 group: ff02::2
          inet6 group: ff02::1
          inet6 group: ff02::1:ff01:c905
          
Iface  6  HWaddr: AE:8D:C2:75:22:DE 
          L2-PDU:1500  MTU:1492  HL:255  RTR  
          Source address length: 6
          Link type: wired
          inet6 addr: fe80::ac8d:c2ff:fe75:22de  scope: link  VAL
          inet6 addr: 2001:9e8:1411:1c00:ac8d:c2ff:fe75:22de  scope: global  VAL
          inet6 group: ff02::2
          inet6 group: ff02::1
          inet6 group: ff02::1:ff75:22de
          
> ping 2600::
ping 2600::
12 bytes from 2600::: icmp_seq=0 ttl=51 time=149.471 ms
12 bytes from 2600::: icmp_seq=1 ttl=51 time=131.725 ms
12 bytes from 2600::: icmp_seq=2 ttl=51 time=132.313 ms

--- 2600:: PING statistics ---
3 packets transmitted, 3 packets received, 0% packet loss
round-trip min/avg/max = 131.725/137.836/149.471 ms
```

let me come up with an alternative variable then.

Reply:
> Heh this does actually work smiley

You seem surprised ^^ I use it regularly for border routers :P

Reply:
> REUSE_TAP=1 maybe?

Sounds good!